getting-started/train_ort.py

```diff
@@ -89,9 +86,6 @@ def evaluate(eval_model, data_source):
     with torch.no_grad():
         for i in range(0, data_source.size(0) - 1, bptt):
             data, targets = get_batch(data_source, i)
-            if len(data) < bptt:
-                print(f"len(data)={len(data)} < {bptt}")
-                continue
             results = trainer.eval_step(data, targets)
```

Comment:
`loss, output = trainer.eval_step(data, taregts)` is more idiomatic to Python. Using outputs as list and then indexing it is too verbose

You can use 

`loss, _= trainer.eval_step(data, taregts)` if you dont care about `output`

The same concept applies to other occurences of `eval_step` and `train_step`

Reply:
Fixed